source/blog/2017-10-10-glimmer-progress-report.md

```diff
@@ -0,0 +1,645 @@
+because components play such a central role, we knew that we _had_ to have a
+tangible implementation for people to play with before we could credibly ask
+them to comment on an RFC. And we knew that having an implementation would
+almost certainly shake out problems with the design that we would need to
+address.
+
+Glimmer.js is our way to iterate on a new component API until we have something
+we feel confident submitting to the Ember RFC process.
+
+We've already received incredibly useful feedback from early adopters. In
+addition, fellow core team member Chad Hietala and I have been on a team at
+LinkedIn using Glimmer.js to build a production application.
+
+[To quote DHH](http://david.heinemeierhansson.com/posts/6-why-theres-no-rails-inc):
+
+> The best frameworks are in my opinion extracted, not envisioned. And the best
+> way to extract is first to actually do.
+
+If you notice that a lot of the items I describe below are performance-related,
+that is at least partly due to our product's near-maniacal focus on mobile
+load times. We are extremely excited about some of the recent breakthroughs
+we've made and have enjoyed proving out some of the more esoteric ideas in a
+real app.
+
+## What's New in Glimmer
+
+### `<my-component />` → `<Component />`
+
+One of the most eagerly-awaited features of Glimmer.js is "angle bracket
+components," or components that you invoke `<like-this />` instead of
+`{{like-this}}`. I personally really like this syntax because it visually
+disambiguates components from the dynamic data that flows through them.
+It also unifies the attribute syntax between HTML and components:
+
+```hbs
+{{! Ember }}
+{{my-button title=title label=(t "Do Something")}}
+
+{{! HTML }}
+<button title={{title}} label={{t "Do Something"}}></button>
+
+{{! Glimmer.js }}
+<my-button @title={{title}} @label={{t "Do Something"}} />
+```
+
+This syntax is how Glimmer.js works today. However, the more we discussed the
+design and started using it in real projects, the more we believed that this
+exact API was flawed and needed to be rethought.
+
+First, a short history lesson. When we introduced components in Ember (back
+before Ember 1.0!), we required them to include a dash (`-`) in their name. This
+rule came from the then-new [Custom Elements spec][custom-elements], a key part
+of the Web Components API.
+
+[custom-elements]: https://www.w3.org/TR/custom-elements/#custom-element-conformance
+
+Web Components have the problem of needing to disambiguate between built-in elements
+and custom elements. What happens if I make a component called `<vr></vr>` and later
+the browser adds a built-in Virtual Reality element with the same name?
+
+The compromise was that custom elements must have a dash, keeping single-word
+elements reserved for future versions of the HTML standard.
+
+Of course, Ember components don't have the same problem, because they use
+`{{`/`}}` as delimeters instead of `<`/`>`. Nonetheless, we preemptively adopted
```

Comment:
When rendered `{{`/`}}` looks confusingly like syntax to me. I had to stare at it for a bit. Perhaps "`{{` and `}}`" "`<` and `>`"

Reply:
👍